BlueDBOnDisk/src/test/java/org/bluedb/disk/ReadableDbOnDiskTest.java

```diff
@@ -64,6 +64,12 @@ public void test_getUntypedCollectionForBackup() throws Exception {
 		} catch (Throwable e) {
 		}
 	}
+    //This test we want to pass into getUntypedCollectionForBackup a string for a filename which does not contain a .meta key_type file
+    //Currently it is causing a null pointer when this happens
+	@Test
+    public void test_getUntypedCollectionForBackup_withInvalidPath() throws BlueDbException {
+        assertNull(db.getUntypedCollectionForBackup("bad_folder"));

```

Comment:
So is this just a folder that doesn't exist at all? We should probably create an empty folder as well that does exist and verify we have the expected result.